server/initializers/migrations/0475-video-file-metadata.ts

```diff
@@ -0,0 +1,24 @@
+import * as Sequelize from 'sequelize'
+
+async function up (utils: {
+  transaction: Sequelize.Transaction,
+  queryInterface: Sequelize.QueryInterface,
+  sequelize: Sequelize.Sequelize
+}): Promise<void> {
+
+  const data = {
+    type: Sequelize.JSON,
```

Comment:
JSONB

Reply:
type is `JSON` in model, should I change all to JSON? It is not like we are going to build queries on it.